{{cookiecutter.project_slug}}/clients/vue3/src/main.css

```diff
@@ -0,0 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import './styles/abstracts/headings.css';
+@import './styles/components/buttons.css';
+@import './styles/components/inputs.css';
```

Comment:
**Info:** I don't believe there's a way to import all of the `*.css` files within a specific folder (ex: `/components/`), so each `.css` file used for more standardized styles have to be imported here separately.

Reply:
So I'm not sure we'd continue to use the styles folder if we're going with tailwind - the styles are mostly used for scss. We may want to keep some of the components files but I would create an index.css file in that folder and import each one and then export all of them so that we can just import all the styles in one line and keep it tidier